cmd/run.go

```diff
@@ -741,6 +765,15 @@ func bindRunFlags(cmd *cobra.Command) {
 	cmd.Flags().Int("datastore-max-cache-size", defaultConfig.Datastore.MaxCacheSize, "the maximum number of cache keys that the storage cache can store before evicting old keys")
 	util.MustBindPFlag("datastore.maxCacheSize", cmd.Flags().Lookup("datastore-max-cache-size"))
 
+	cmd.Flags().Int("datastore-max-open-conns", defaultConfig.Datastore.MaxOpenConns, "the maximum number of open connections to the datastore")
```

Comment:
These currently mirror the config names in database/sql, and I think in English "max open connections" and "connection max idle time" sound natural. Not so much "connections max open". I would prefer to keep them as it, but my mind can be changed.

Reply:
> These currently mirror the config names in database/sql, and I think in English "max open connections" and "connection max idle time" sound natural. Not so much "connections max open".

The community using the flags of our CLI don't care so much about `database/sql` package of Go, so though it maps nicely to the configs exported out of that package, I don't think that's a compelling reason to name them that way.

I do agree it sounds more natural to say `datastore-max-open-conns`, but the objective behind the names and structure of the CLI flags is to organize by category and sub-category etc.. Since these configs deal with datastore connection settings, I think it makes sense to follow the other practices established herein and follow the structure `datastore-conn-max-open` and `datastore-conn-max-idle` because we're configuring the subcategory of datastore specific settings specifically for connection related settings. `max-idle`,  `max-idle-time`, `max-open`, `max-lifetime` are all settings specific to `datastore-conn` subcategory of settings.

I suppose I'm not strongly held to the opinion above, but we would be following status quo established herein a little better if we organized by sub-category of settings.

Reply:
```
datastore-max-open-conns
datastore-max-idle-conns
datastore-conn-max-idle-time
datastore-conn-max-lifetime
```

The first two deal with connections (plural), the second two deal with one connection. I think it's fine the way it is